content/post/agile-analytics-p2.md

```diff
@@ -0,0 +1,60 @@
+
+As a researcher, I often find the most valuable insights I generate come from investigating something that looked odd or interesting and was only tangentially related to the task at hand. Exploratory analysis makes up the "science" part of data science and we want to ensure that our Analysts have the time and the inclination to pursue these opportunities and generate novel insights.
+
+
+## Exploratory data analysis
+While exploratory analysis can start from a fortuitous finding, it can also start from general curiosity, a specific hypothesis, or the belief that no one has looked at this before, so there must me insights to be found. Exploratory data analysis is an important part of the "basic research" function of an Analytics or Data Science team. 
+
+Unfortunately, it is very hard to write "user stories" or "acceptance criteria" for this sort of work because ... the whole point is that you are not sure what you are going to find, and it requires a researcher's intuition to know when the story is "done" (because there are always more hypotheses you *could* look into, but often those marginal investigations are not valuable).
+
+The nature of exploratory data analysis means that the objectives of the analysis may change *as you do the work*. You sometimes realize "oh x is not actually very interesting -- the real issue is understanding y." If your scrum process is too inflexible, you can end up expending effort against tasks that are not valuable simply because they are required to get the story done. Alternatively, you can write acceptance criteria so broad (to allow for flexibility down the road) that they becomes meaningless.  While (software) engineering is a discipline of creatively applying mathematical tools to a well-defined problem, data science is about creatively applying mathematical tools to *poorly defined* problems, and it is this fundamental difference in nature that makes partitioning the work into small well-scoped tasks sometimes impossible for data science.
+
+## Product ownership / story-writing
+
+When you are trying to build a model of customer churn, there is some amount of judgment and art that goes into doing good data science. The acceptance criteria of "predict which customers are going to churn" is too vague to be helpful and does not capture the art of investigating a problem by poking at it from multiple sides.  Additionally, it can be very difficult for someone external to write stories or be the product owner for the team. Often, the Data Scientists or Analysts are *the* domain experts in the company and there is not anyone else who is better informed about what the next step in a research program should be. 
```

Comment:
Should "are" be italicized instead of "the"?

Reply:
Stet! :)